homework-1/src/Shape.ts

```diff
@@ -0,0 +1,43 @@
+import { Point } from './Point';
+
+export const DEFAULT_COLOR = 'green';
+
+export abstract class Shape {
+    protected color: string;
+    protected filled: boolean;
+    protected points: Point[];
+
+    constructor(points: Point[]);
+    constructor(points: Point[], color?: string, filled?: boolean) {
```

Comment:
According to the Hometask "color" and "filled" should be obligatory for this overloading

Reply:
Fixed, thanks